### PR TITLE
Fix layout on subscribers page

### PR DESCRIPTION
**Pull request by kylefiedler**
screenshot 2015-11-24 14 58 09

### CODE REVIEW

**Comment by schneems** [comment]
Good catch. That page looks much better. On a separate issue I noticed when you view the main repo site on a mobile you get a phone full of huge faces

**Comment by kylefiedler** [comment]
@schneems can you put that in a separate issue?

File: app/views/users/show.html.slim [hunk 1]
```diff
@@ -8,9 +8,9 @@
       h1.subpage-primary-title= @user.github
       h2.subpage-secondary-title.user-email= @user.email
 
-      .user-actions
-        = link_to 'Delete Account'.html_safe, user_path(@user), method: "delete", confirm: "#{t("web-app-theme.confirm", default: "Are you sure?")}", class: 'delete-account'
-        = link_to 'Edit Account'.html_safe, edit_user_path(@user), class: 'edit-account'
+      .subpage-header-actions
+        = link_to 'Delete Account'.html_safe, user_path(@user), method: "delete", confirm: "#{t("web-app-theme.confirm", default: "Are you sure?")}"
+        = link_to 'Edit Account'.html_safe, edit_user_path(@user), class: 'header-primary-action'
```
Comment (ethangl) [inline app/views/users/show.html.slim]: Single quotes??!?

File: app/views/subscribers/show.html.slim [hunk 2]
```diff
@@ -1,4 +1,10 @@
-.repo-wrapper
-  h2.repo-section-title Subscribers to #{link_to @repo.path, @repo}
+div class="subpage-content-wrapper #{ @repo.weight }"
+  header.subpage-content-header
+    h1.subpage-primary-title #{ @repo.subscribers.count } #{ @repo.name } Subscribers
+    .subpage-header-actions
+      = link_to @repo, class: "header-primary-action"
+        | Back to 
+        = @repo.name
```
Comment (ethangl) [inline app/views/subscribers/show.html.slim]: `= link_to "Back to #{@repo.name}", @repo, class: "header-primary-action"`?

**Comment by ethangl** [comment]
:nail_care:

**Pull request merged**